subsys/zigbee/lib/zigbee_fota/src/zigbee_fota.c

```diff
@@ -431,7 +400,7 @@ zb_uint8_t zb_ota_process_chunk(const zb_zcl_ota_upgrade_value_param_t *ota,
 
 	/* Process image header and save it in the memory. */
 	if (ota_ctx.mandatory_header_finished == false) {
```

Comment:
`if (!ota_ctx.mandatory_header_finished) {` is more natural

Reply:
Changed.